modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/RelatedObjectEntryOpenAPIContributor.java

```diff
@@ -145,14 +146,45 @@ private void _contribute(
 			_getIdParameterTemplate(schemaName), StringPool.SLASH,
 			systemObjectRelationship.getName());
 
+		Schema parameterSchema = null;
```

Comment:
Maybe better to use a more descriptive name, for example idSchema ? (I am not sure, but I think Brian will like more) xD